web/libs/editor/src/components/Comments/Comment/CommentItem.tsx

```diff
@@ -0,0 +1,211 @@
+          </Tooltip>
+        </Elem>
+      );
+    return null;
+  };
+
+  return (
+    <Block name="comment-item" mod={{ resolved }}>
+      <Space spread size="medium" truncated>
+        <Space size="small" truncated>
+          <Elem tag={Userpic} user={createdBy} name="userpic" showUsername username={createdBy} />
+          <Elem name="name" tag="span">
+            {userDisplayName(createdBy)}
+          </Elem>
+        </Space>
+
+        <Space size="small">
+          <Elem name="resolved" component={IconCheck} />
+          <Elem name="saving" mod={{ hide: isPersisted }}>
+            <Elem name="dot" />
+          </Elem>
+          <TimeTracker />
+        </Space>
+      </Space>
+
+      <Elem name="content">
+        <Elem name="text">
+          {isEditMode ? (
+            <CommentFormBase
+              value={text}
+              onSubmit={async (value) => {
+                await updateComment(value);
+                setText(value);
+                await listComments({ suppressClearComments: true });
+              }}
+            />
+          ) : isConfirmDelete ? (
+            <Elem name="confirmForm">
+              <Elem name="question">Are you sure?</Elem>
+              <Elem name="controls">
+                <Button onClick={() => deleteComment()} size="compact" look="danger" autoFocus>
+                  Yes
+                </Button>
+                <Button onClick={() => setConfirmMode(false)} size="compact">
+                  No
+                </Button>
+              </Elem>
+            </Elem>
+          ) : (
+            <>
+              {text}
+              {hasLinkState && (
+                <Elem name="linkState">
+                  <LinkState linking={linking} region={region} />
+                </Elem>
+              )}
+            </>
+          )}
+        </Elem>
+
+        <Elem
+          name="actions"
+          onClick={(e: any) => {
+            e.stopPropagation();
+            e.preventDefault();
+          }}
+        >
+          {isPersisted && (isCreator || canResolveAny) && (
+            <Dropdown.Trigger
+              content={
+                <Menu size="auto">
+                  <Menu.Item onClick={toggleResolve}>{resolved ? "Unresolve" : "Resolve"}</Menu.Item>
+                  {isCreator && (
+                    <>
+                      <Menu.Item
+                        onClick={() => {
+                          const isGoingIntoEditMode = !isEditMode;
+
+                          setEditMode(isGoingIntoEditMode);
+                          if (!isGoingIntoEditMode) {
+                            setText(initialText);
+                          }
+                        }}
+                      >
+                        {isEditMode ? "Cancel edit" : "Edit"}
+                      </Menu.Item>
+                      <Menu.Item onClick={toggleLink}>{regionRef?.region ? "Unlink" : "Link to ..."}</Menu.Item>
```

Comment:
```suggestion
                      <Menu.Item onClick={toggleLink}>{regionRef?.region ? "Unlink" : "Link to..."}</Menu.Item>
```
dialog items in menus should have ellipsis right after the text